src/console.rs

```diff
@@ -87,12 +87,12 @@ pub(crate) unsafe fn init(image_base: usize) -> anyhow::Result<()> {
     LateStatic::assign(
         &S,
         State {
-            console_context: transmute(image_base + 0x2f000f0),
-            print_to_console: transmute(image_base + 0x85c290),
+            console_context: transmute(image_base + 0x2F9A800),//1.5.97: 0x2f000f0 -(score 1.0)-> 1.6.318: 0x2F9A800 -> addressLib ID: 401203 -> 1.6.323: 0x2f9a800
+            print_to_console: transmute(image_base + 0x8893c0),//1.5.97: 0x85c290 -(score 0.955)-> 1.6.318: 0x889650 -> addressLib ID: 51109 -> 1.6.323: 0x8893c0
```

Comment:
what are these scores?

Reply:
I used the file called [Attempt at matching 1.5.97.0 to 1.6.318.0](https://www.nexusmods.com/skyrimspecialedition/mods/32444?tab=files) found in the Miscellaneous files section where there's a list of IDs from both Versions as well as a score of how well the code behind those offsets matches between the versions. I then created address dumps for runtime versions 1.6.318 and 1.6.323 with the offsets and addressLibrary IDs for each and used that to find the offsets for 1.6.323. the scores are basically just a mental note where it might be most important to look should something go wrong (lower score -> lower likelyhood of it working 100%). I tested everything (I think) but there may still be situations where it goes wrong and the scores may be a hint where to look.